debian/rules

```diff
@@ -43,15 +43,15 @@ binary-syncd-rpc:
 #	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
 
 override_dh_auto_configure:
+    ifeq ($(shell cat /tmp/syncd-build), --enable-rpcserver=yes)
```

Comment:
run sed command against copied version? If it has wrong value it would be overwritten. If it has right value it would be skipped.
What do you think?